gourmet/OptionParser.py

```diff
@@ -35,8 +34,8 @@
     argcomplete.autocomplete(parser)
 
 try:
-    args = parser.parse_args()
-    print('args1 = %s' % args)
+    args = parser.parse_known_args()[0]
```

Comment:
This in an important change! 

Note, that this `OptionParser` is heavily used in man imports. And that's the reason why from now on we take known arguments, and pass unknown ones. But we don't fail if something else is passed — probably that additional _something_ belongs to someone else. Say, it might be `nosetest` argument.